### PR TITLE
feat: Add retry log

### PR DESCRIPTION
**Pull request by disq**
Cleaned up

### CODE REVIEW

File: client/retry.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	}
+}
+
+func (r *retryer) RetryDelay(attempt int, err error) (time.Duration, error) {
+	dur, err := r.backoff.BackoffDelay(attempt, err)
+	if err != nil {
+		r.logger.Debug("retryDelay returned err", "err", err, "duration", dur.String())
+	} else {
+		r.logger.Debug("retryDelay will wait", "duration", dur.String())
+	}
```
Comment (roneli) [inline client/retry.go:40]: extract resource type from error

**Review by roneli — CHANGES_REQUESTED** [review]

File: client/retry.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+				o.MaxBackoff = maxbackoff
+				o.Backoff = backoff
+			}),
+			backoff: backoff,
+			logger:  logger,
+		}
+	}
+}
+
+func (r *retryer) RetryDelay(attempt int, err error) (time.Duration, error) {
```
Comment (irmatov) [inline client/retry.go:34]: Why not call Standard.RetryDelay here?

Comment (disq) [reply client/retry.go:30]: Fixed 🙈

File: client/retry.go (hunk 1 above)
Comment (disq) [reply client/retry.go]: I tried

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**